packages/repack/src/commands/start.ts

```diff
@@ -172,16 +172,18 @@ export async function start(_: string[], config: Config, args: StartArguments) {
           },
         },
         api: {
-          getPlatforms: async () => Object.keys(compiler.workers),
-          getAssets: async (platform) =>
-            Object.entries(compiler.assetsCache[platform] ?? {}).map(
-              ([name, asset]) => ({
-                name,
-                size: asset.info.size,
-              })
+          getPlatforms: () => Promise.resolve(Object.keys(compiler.workers)),
+          getAssets: (platform) =>
+            Promise.resolve(
+              Object.entries(compiler.assetsCache[platform] ?? {}).map(
+                ([name, asset]) => ({
+                  name,
+                  size: asset.info.size,
+                })
+              )
             ),
-          getCompilationStats: async (platform) =>
-            compiler.statsCache[platform] ?? null,
+          getCompilationStats: (platform) =>
+            Promise.resolve(compiler.statsCache[platform] ?? null),
```

Comment:
changed `async` to `Promise.resolve` to satisfy the requirements, otherwise TS & eslint have a conflict